pinecone/grpc/pinecone.py

```diff
@@ -117,15 +118,17 @@ def Index(self, name: str = '', host: str = '', **kwargs):
         index.query(vector=[...], top_k=10)
         ```
         """
-        if name == '' and host == '':
+        if name == "" and host == "":
```

Comment:
Building off of @ssmith-pc's wisdom, plz remove linting changes from your PRs :)

Reply:
We should be running `black` on all commits to avoid this issue.